log/log.go

```diff
@@ -265,3 +268,31 @@ func Warnw(msg string, keyvalues ...interface{}) {
 func Errorw(err error, msg string) {
 	Logger().Error().Err(err).Msg(msg)
 }
+
+// prettyStack returns a pretty stack trace
+//
+// if disableStackTrace is true (env LOG_DISABLE_STACKTRACE=true), it returns immediately
+func prettyStack() string {
```

Comment:
@mvdan can you review this func?

Reply:
@altergui did you see https://pkg.go.dev/runtime/debug#Stack? I think that's basically what we want. It doesn't skip any callers, but honestly I don't think that's a big deal, particularly if you can avoid `func prettyStack` to begin with.

Reply:
that was indeed my initial one-liner approach https://github.com/vocdoni/vocdoni-node/commit/1edb30acee08e7edb0525e26df8f547e75149a6b
but the log_test failed because of the new output, so i did the prettyStack to skip the `0xfff` thinking that would give me a reproduceable, testable output. only to THEN notice that the paths printed also depend on the host. :facepalm: 

happy to go back to the one-liner trivial-to-review PR :grin: